fix from upstream git repo

From 9aab81145a6ce26272b198419d288a0417e683c5 Mon Sep 17 00:00:00 2001
From: Jouni Malinen <j@w1.fi>
Date: Sat, 15 Feb 2014 11:49:09 +0200
Subject: [PATCH] Fix nt_password_hash build

This needs librt on some systems for clock_gettime().

Signed-off-by: Jouni Malinen <j@w1.fi>
---
 hostapd/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/hostapd/Makefile b/hostapd/Makefile
index 5fd6481..ef093e5 100644
--- a/hostapd/Makefile
+++ b/hostapd/Makefile
@@ -95,6 +95,7 @@ ifeq ($(CONFIG_ELOOP), eloop)
 LIBS += -lrt
 LIBS_c += -lrt
 LIBS_h += -lrt
+LIBS_n += -lrt
 endif
 
 OBJS += ../src/utils/common.o
-- 
1.9.2

